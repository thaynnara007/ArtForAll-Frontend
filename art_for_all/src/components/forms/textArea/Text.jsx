import React from 'react';

class Text extends React.Component{
    render(){
        return(
            <div className="form-group">
                <textarea className="form-control" placeholder={this.props.placeHolder} onChange={(event, data) => this.props.handleChange(event, this.props.data)}></textarea>
            </div>
        )
    }
}

export default Text;