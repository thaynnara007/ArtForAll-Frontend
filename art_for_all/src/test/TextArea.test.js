import React from 'react';
import Jest from 'jest';
import Enzyme from './Enzyme';
import {shallow} from 'enzyme';
import Text from '../components/forms/textArea/Text';

describe("TEXT AREA - Testing with snapshots", () =>{

    it('TEST01: it shoul render correctly', () =>{
        const text = shallow(<Text></Text>);
        expect(text).toMatchSnapshot();
    })

    it('TEST02: it shoul render correctly with placeHolder=hello there', () =>{
        const text = shallow(<Text placeHolder="hello there"></Text>);
        expect(text).toMatchSnapshot();
    })
})