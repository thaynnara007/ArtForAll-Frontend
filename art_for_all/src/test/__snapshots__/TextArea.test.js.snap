// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TEXT AREA - Testing with snapshots TEST01: it shoul render correctly 1`] = `
<div
  className="form-group"
>
  <textarea
    className="form-control"
    onChange={[Function]}
  />
</div>
`;

exports[`TEXT AREA - Testing with snapshots TEST02: it shoul render correctly with placeHolder=hello there 1`] = `
<div
  className="form-group"
>
  <textarea
    className="form-control"
    onChange={[Function]}
    placeholder="hello there"
  />
</div>
`;
